game.go

```diff
@@ -24,10 +29,64 @@ func (game *Game) changeTurn() {
 }
 
 func (game *Game) placePiece(row int, col int, player Player) bool {
-    if row > game.gameBoardSize || col > game.gameBoardSize {
-        fmt.Printf("Error, the row and column must be less than or equal to %v to fit on the game board. Please try again.\n", game.gameBoardSize)
+    if !game.indexIsOnBoard(row - 1, col - 1) {
+        fmt.Printf("Error, the row and column must be positive and less than or equal to %v to fit on the game board. Please try again.\n", game.gameBoardSize)
         return false
     }
     game.board[row - 1][col - 1] = uint8(player)
     return true
 }
+
+func (game *Game) checkForCapturedPieces() bool {
+    opposingPlayer := game.currentPlayer.opposingPlayer()
+    return game.checkForCapturedPiecesOfPlayer(opposingPlayer)
```

Comment:
I made this a separate function because we also want to use `.checkForCapturedPiecesOfPlayer()` to check if the current player has any pieces captured after their own move, which would invalidate the move.